saathi.sql

```diff
@@ -63,7 +63,8 @@ CREATE TABLE IF NOT EXISTS `contact_details` (
   `email` varchar(255) NOT NULL,
   `mailing_list` varchar(255) NOT NULL,
   `mailing_address` varchar(255) NOT NULL,
-  `gis_url` varchar(255) NULL,
+  `longitude` varchar(255) NULL,
+  `latitude` varchar(255) NULL,
```

Comment:
I think it would be better if we store them separately but do you want me to use POINT? Is there any disadvantage if we store them separately?